src/mongo/mod.rs

```diff
@@ -18,6 +48,39 @@ impl Image for Mongo {
     fn ready_conditions(&self) -> Vec<WaitFor> {
         vec![WaitFor::message_on_stdout("Waiting for connections")]
     }
+
+    fn cmd(&self) -> impl IntoIterator<Item = impl Into<std::borrow::Cow<'_, str>>> {
+        match self.kind {
+            InstanceKind::Standalone => Vec::<String>::new(),
+            InstanceKind::ReplSet => vec!["--replSet".to_string(), "rs".to_string()],
+        }
+    }
+
+    fn exec_after_start(
+        &self,
+        _: testcontainers::core::ContainerState,
+    ) -> Result<Vec<ExecCommand>, testcontainers::TestcontainersError> {
+        match self.kind {
+            InstanceKind::Standalone => Ok(Default::default()),
+            InstanceKind::ReplSet => Ok(vec![
+                ExecCommand::new(vec![
+                    "mongosh".to_string(),
+                    "--quiet".to_string(),
+                    "--eval".to_string(),
+                    "'rs.initiate()'".to_string(),
+                ])
+                .with_cmd_ready_condition(CmdWaitFor::message_on_stdout(
+                    "Using a default configuration for the set",
+                )),
+                ExecCommand::new(vec![
+                    "mongosh".to_string(),
+                    "--eval".to_string(),
+                    "'rs.status()'".to_string(),
+                ])
+                .with_cmd_ready_condition(CmdWaitFor::message_on_stdout("ok: 1")),
```

Comment:
I think `rs.status` doesn't wait for "ok", right?

In that case we probably need to wrap it into something like:

```
until mongosh --eval 'rs.status()'; do sleep 1; done
```

WDYT?